mobile/src/views/VerificationView.jsx

```diff
@@ -48,6 +48,21 @@ const VerifcationView = ({navigation}) => {
       })
       .then(data => {
         console.log(data);
```

Comment:
Extra console log

Reply:
@yismailuofa, just to clarify, we want to remove extra console.logs?

Reply:
Removed extra console.log